pycheribuild/projects/cross/ros2.py

```diff
@@ -63,10 +66,31 @@ def _run_colcon(self, **kwargs):
         if self.config.verbose:
             cmdline.append("--event-handlers")
             cmdline.append("console_cohesion+")
-        return self.run_cmd(cmdline, cwd=self.sourceDir, **kwargs)
+        self.run_cmd(cmdline, cwd=self.sourceDir, **kwargs)
+
+    def _find_file(self, name, path):
+        for root, dirs, files in os.walk(path):
```

Comment:
```suggestion
        for root, dirs, files in os.walk(str(path)):
```
Seems to be needed for python 3.5